src/sentry/api/endpoints/group_autofix_setup_check.py

```diff
@@ -118,6 +121,23 @@ def get(self, request: Request, group: Group) -> Response:
                 "repos": repos,
             }
 
+        # Check the current user first using PromptsActivity
+        user_has_acknowledged = PromptsActivity.objects.filter(
+            user_id=request.user.id,
+            feature=feature_name,
+            organization_id=org.id,
+            project_id=0,
+        ).exists()
```

Comment:
why is `project_id` 0? Is that equivalent to saying the project doesn't matter?

Reply:
I'm guessing this is the same stuff behind the UI tours?

Reply:
this is prompts which are similar to tours/guides. project=0 just means its not project specific https://github.com/getsentry/sentry/blob/df97eb1045122773d6efe0d11df24cb2bb042646/src/sentry/api/endpoints/prompts_activity.py#L96-L97